<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginWidget</class>
 <widget class="QWidget" name="PluginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxPlugin">
     <property name="toolTip">
      <string>Enable this clipboard content to be saved and displayed</string>
     </property>
     <property name="text">
      <string>&amp;Enable</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAuthor">
     <property name="text">
      <string>author: %1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
